### PR TITLE
pkp/pkp-lib#2163 Update submissions guidance to match new submission …

### PR DESCRIPTION
**Pull request by NateWr**
…lists

### CODE REVIEW

File: en/submissions.md [hunk 1]
```diff
@@ -13,21 +13,21 @@ Editorial and administrative users also have access to all published and archive
...
-Other editorial or administrative staff can view the **My Assigned** panel, which shows any submissions currently assigned to them. You can keep track of what stage your assigned submissions are in.
+## <a name="unassigned"></a>Unassigned
 
-All users can view the **My Authored** panel, which includes only your own submissions.
+Journal Managers can view any new submissions which have not yet been assigned an editor or section editor. Use this tab to ensure each submission has an editor assigned to manage the workflow.
 
 ## <a name="active"></a>Active
 
-Find any submission in the system that hasn't been published or declined. Only editorial and administrative users have access to this tab.
+Find any submissions in the system that havn't been published or declined. Only editorial and administrative users have access to this tab.
```
Comment (asmecher) [inline en/submissions.md]: *haven't

**Comment by asmecher** [comment]
One tiny typo, otherwise this looks good. @mtub, in this case would you rather that we removed the key from other locales, or left it slightly incorrect?

**Comment by NateWr** [comment]
Cheers, fixed. :+1:

**Comment by NateWr** [comment]
@asmecher I've refreshed this with a few more features added recently. Take one last look?

**Comment by asmecher** [comment]
Looks good to me, @NateWr. @stranack, would you like to be notified when changes like this are made?

**Comment by NateWr** [comment]
I reached out to @stranack over Slack and he gave the :+1:.

**Pull request merged**